legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/testSuite/dynaFunctions/string.pure

```diff
@@ -369,4 +369,18 @@ function <<paramTest.Test>> meta::relational::tests::dbSpecificTests::sqlQueryTe
   let dynaFunc = ^DynaFunction(name='ascii', parameters=[^Literal(value='a')]);
   let expected = ^Literal(value=97);
   runDynaFunctionDatabaseTest($dynaFunc, $expected, $config);
+}
+
+function <<paramTest.Test>> meta::relational::tests::dbSpecificTests::sqlQueryTests::dynaFunctions::jaroWinklerSimilarity::testJaroWinklerSimilarity(config:DbTestConfig[1]):Boolean[1]
+{
+  let dynaFunc = ^DynaFunction(name='jaroWinklerSimilarity', parameters=[^Literal(value='John Smith'), ^Literal(value = 'Jane Smith')]);
+  let expected = ^Literal(value=88);
```

Comment:
Wasn't it 0.88 in legend-pure test and 88 here. Its strange to have similarity values outside the range [0, 1] or [-1, 1]

Reply:
https://docs.snowflake.com/en/sql-reference/functions/jarowinkler_similarity snowflake returns 0 to 100.

Reply:
As discussed offline, will change snowflake command to return range [0, 1]